docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -470,24 +470,70 @@ Collection versions use `Semantic Versioning <https://semver.org/>`_ for version
 
 .. _migrate_to_collection:
 
-Migrating Ansible content to a collection
-=========================================
+Migrating Ansible content to a different collection
+====================================================
+
+To migrate content from one collection to another, you need to create three PRs as follows:
+
+#. Create a PR against the old collection to remove the content.
+#. Create a PR against the new collection to add the files removed in step 1.
+#. Update the ``ansible/ansible:devel`` branch entries for all files moved.
+
+
+Removing the content from the old collection
+----------------------------------------------
+
+Create a PR against the old collection repo to remove the modules, module_utils, plugins, and docs_fragments related to this migration:
+
+#. If you are moving an action plugin, include the corresponding module with documentation.
+#. If you are moving a module, move any corresponding action plugin that should stay with it.
+#. Remove any entries about removed plugins from ``meta/action_groups.yml`` and ``meta/runtime.yml``. Ensure they are added into the new repo.
```

Comment:
```suggestion
#. Remove any entries about removed plugins from ``meta/runtime.yml``. Ensure they are added into the new repo.
```
Action groups are now defined inside `meta/runtime.yml` as well. These entries cannot be moved that simply though, since action groups are currently limited to a certain hard-coded list of group(s) allowed per collection. (It's best if modules mentioned in action groups are not moved at all.)